infrastructure/quick-lint-js-web/apache2/mods-available/fcgid.conf

```diff
@@ -0,0 +1,9 @@
+<IfModule mod_fcgid.c>
```

Comment:
Is this file used? I don't see an `a2enconf fcgid`.

Reply:
Yes, it's being used. we are overwriting the file.